scripts/setup-test-data/src/setup_test_data/cli.py

```
@@ -0,0 +1,410 @@
+    prerelease: bool
+    body: Optional[str]
+    published_at: str
+    assets: list[dict]
+
+    def __post_init__(self) -> None:
+        if isinstance(self.id, str):
+            self.id = int(self.id)
+
+
+def determine_release_type(
+    variant: str, tag_name: str, prerelease: bool
```

Ruff (0.15.15)

[warning] Boolean-typed positional argument in function definition

(FBT001)